checker/src/org/checkerframework/checker/signedness/SignednessAnnotatedTypeFactory.java

```diff
@@ -154,5 +174,42 @@ public Void visitCompoundAssignment(CompoundAssignmentTree tree, AnnotatedTypeMi
             annotateBoolean(type);
             return null;
         }
+
+        @Override
+        public Void visitIdentifier(IdentifierTree tree, AnnotatedTypeMirror type) {
+            TypeMirror javaType = type.getUnderlyingType();
+            ValueAnnotatedTypeFactory valFact = getValueAnnotatedTypeFactory();
+            Range treeRange = IndexUtil.getPossibleValues(valFact.getAnnotatedType(tree), valFact);
+
+            if (treeRange != null) {
+                switch (javaType.getKind()) {
+                    case BYTE:
```

Comment:
This only handles primitives, but the Value Checker handles wrappers such as `Integer`.  Please support them or write a comment explaining why they are not supported.